test/DataFileUtil_server_test.py

```diff
@@ -1447,12 +1449,23 @@ def test_download_direct_link_with_no_content_disposition(self):
             'download_type': 'Direct Download',
             'file_url': file_url
         }
-
         ret1 = self.impl.download_web_file(self.ctx, params)[0]
         self.assertIsNotNone(ret1['copy_file_path'])
         self.assertEqual(os.path.basename(ret1['copy_file_path']),
                          'SP1.fq')
 
+    def test_download_direct_link_web_file(self):
+        # direct download link of 'file1.txt'
```

Comment:
I would add a comment like "Content-Length for this file is None". And change the func name to something like test_download_direct_link_none_content_length. Other than that, I think it's good to go.